cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/Poll.hs

```diff
@@ -34,21 +37,20 @@ import           System.IO (stderr, stdin, stdout)
 
 runGovernancePollCmds :: GovernancePollCmds era -> ExceptT GovernanceCmdError IO ()
 runGovernancePollCmds = \case
-  GovernanceCreatePoll w prompt choices nonce out ->
-    runGovernanceCreatePoll w prompt choices nonce out
-  GovernanceAnswerPoll w poll ix mOutFile ->
-    runGovernanceAnswerPoll w poll ix mOutFile
-  GovernanceVerifyPoll w poll metadata mOutFile ->
-    runGovernanceVerifyPoll w poll metadata mOutFile
+  GovernanceCreatePoll _ prompt choices nonce out ->
+    runGovernanceCreatePoll prompt choices nonce out
+  GovernanceAnswerPoll _ poll ix mOutFile ->
+    runGovernanceAnswerPoll poll ix mOutFile
+  GovernanceVerifyPoll _ poll metadata mOutFile ->
+    runGovernanceVerifyPoll poll metadata mOutFile
```

Comment:
Can you pass in the `w` to the run functions as before?

Reply:
Done :+1: